source/particles/particle_handler.cc

```diff
@@ -177,7 +180,6 @@ namespace Particles
     for (auto &particle : *this)
       {
         particle.set_property_pool(*property_pool);
-        particle.set_properties(from_particle->get_properties());
```

Comment:
Is this a mistake? It seems to me like you're no longer copying the properties now. Or am I missing something?

Reply:
Since #11111 `set_property_pool` also copies existing properties, so this call is not longer needed.

Reply:
Ah, I see. Thanks for the hint!